spec/user_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
+
+  subject do
+    user = User.new(name: 'Tom', photo: 'image.png', bio: 'I am programmer', posts_counter: 0)
+    Post.new(title: 'Hello world', text: 'Helloworld', likes_counter: 0, comments_counter: 0, user_id: user.id)
+  end
+
+  before { subject.save }
+
+  it 'should check the validation' do
+    expect(subject).to be_valid
+  end
+end
```

Comment:
- [ ] Kindly add more specific tests for the respective `name` and `posts_counter` validations.

- [ ] Custom methods should also be tested. Kindly add test for the `three_recent_posts` method in the User model.

- [ ] Your test in this file is failing. Kindly fix that.

  ![image](https://user-images.githubusercontent.com/36542986/221371719-8e6dc72b-66bb-4929-b8a7-61600b3781ac.png)